lib/views/sbsearch.js

```diff
@@ -3,6 +3,10 @@ var pug = require('pug')
 const config = require('../config')
 
 module.exports = async function (req, res) {
+  res.setHeader('Cache-Control', 'no-cache')
+  res.setHeader('Cache-Control', 'no-store')
+  res.setHeader('Pragma', 'no-cache')
```

Comment:
What're these about?

Reply:
This is fixing a browser bug for sequence-search.  Was not supposed to be part of this PR.